src/css/wizard-navigation-bar.scss

```diff
@@ -0,0 +1,394 @@
+// Temporary variables used to pass parameters from the `define-style` mixin to its content block.
+// They will no longer be needed after content arguments support is released:
+//
+// - https://github.com/sass/sass/issues/871
+// - https://github.com/sass/libsass/releases
+// - https://github.com/sass/node-sass/releases/
+//
+// Using global variables is the recommended workaround
+//
+// - https://github.com/sass/sass/issues/626#issuecomment-345432420
+// - https://github.com/sass/sass/issues/871
+
+// indicator state: 'default' or 'hover'
+$wz-indicator-state_: null !global;
```

Comment:
I wanted to distinguish these variables (which are kind of 'temporary' and used for passing parameters to content blocks) from ordinary ones (declared in `archwizard.scss` and meant for user to override). By adding an underscore, I am trying to say that these variables are kind of 'internal' or 'protected'.

This is not a common convention (at least I did not refer to any when I named the variables), so I will have no objections if you think that underscores here are unnecessary and should be removed to make the code a bit cleaner.

Reply:
I agree that marking the variables makes sense in this case, but I would suggest to change the marking style, because the `_` at the end looks like insider-knowledge which an arbitrary reader is unlikely to have. Maybe we can give said variables a prefix, which makes it clear that these variables are for internal use? (i.e. something like `$internal-wz-indicator-state`)

Reply:
These variables are not completely internal. In theory, as this PR allows users to reuse the provided SCSS, a user can add a custom style. And he/she will need these variables to implement it. I would feel awkward if I had to access some variables explicitly marked as "internal".

Does it make sense? Are there any other naming patterns you can think of? Perhaps removing the underscores completely will be the easiest choice.

Reply:
An important question is: will a user override the `scss` file(s) or the created/generated `css` file? I mean the css file doesn't contain the variables anymore. So if a user normally overrides the `css` files I think we can simply remove the `_` without replacing it. Otherwise it may be a better idea to mark the variables somehow.

Reply:
I think, many will opt for  `scss`, because it allows to customize color scheme just by defining variables.

How about renaming the variables from 

```
$wz-indicator-state_
$wz-indicator-color_
```

to 

```
$wz-param-indicator-state
$wz-param-indicator-color
```

?

The "param" here indicates that this variables is basically a mixin parameter.

Reply:
I'm fine with that

Reply:
Done in c1fadda